### DISCUSSION TITLE
Welcome to sarama discussions!

### DISCUSSION THREAD

**Discussion by dnwe** [IBM/sarama#2493] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by haunt98** [comment]
Anyone having context why sarama migrate from shopify to IBM?

**Reply by mworzala** [reply]
https://github.com/IBM/sarama/issues/2461
